src/pages/ReimbursementAccount/ACHContractStep.js

```diff
@@ -36,74 +38,111 @@ const propTypes = {
 class ACHContractStep extends React.Component {
     constructor(props) {
         super(props);
+        this.validate = this.validate.bind(this);
 
         this.addBeneficialOwner = this.addBeneficialOwner.bind(this);
         this.submit = this.submit.bind(this);
 
         this.state = {
             ownsMoreThan25Percent: ReimbursementAccountUtils.getDefaultStateForField(props, 'ownsMoreThan25Percent', false),
             hasOtherBeneficialOwners: ReimbursementAccountUtils.getDefaultStateForField(props, 'hasOtherBeneficialOwners', false),
-            acceptTermsAndConditions: ReimbursementAccountUtils.getDefaultStateForField(props, 'acceptTermsAndConditions', false),
-            certifyTrueInformation: ReimbursementAccountUtils.getDefaultStateForField(props, 'certifyTrueInformation', false),
             beneficialOwners: ReimbursementAccountUtils.getDefaultStateForField(props, 'beneficialOwners', []),
         };
-
-        // These fields need to be filled out in order to submit the form (doesn't include IdentityForm fields)
-        this.requiredFields = [
-            'acceptTermsAndConditions',
-            'certifyTrueInformation',
-        ];
-
-        // Map a field to the key of the error's translation
-        this.errorTranslationKeys = {
-            acceptTermsAndConditions: 'common.error.acceptedTerms',
-            certifyTrueInformation: 'beneficialOwnersStep.error.certify',
-        };
-
-        this.getErrors = () => ReimbursementAccountUtils.getErrors(this.props);
```

Comment:
It looks like `ReimbursementAccountUtils.getErrors`, `getErrorText`, `clearError`, `clearErrors` are no longer used, so we can delete those functions in `ReimbursementAccountUtils.js`